tests/unit/dataactvalidator/test_c23_award_financial_1.py

```diff
@@ -14,55 +14,86 @@ def test_column_headers(database):
 
 
 def test_success(database):
-    """ Test that a four digit object class with no flag is a success, and a three digit object class with
-        a flag is a success. Ignores rows with parent_award_id from AwardFinancialFactory and doesn't care about
-        parent_award_id in AwardProcurementFactory """
+    """ Test For each unique PIID in File C, the sum of each TransactionObligatedAmount should match (but with opposite
```

Comment:
would be helpful to understand tests if comment included that Award Financial entries that have a parent id , will be ignored.

Reply:
Fixed